src/components/services/Radarr.vue

```diff
@@ -0,0 +1,153 @@
+<template>
+  <div>
+    <div class="card" :class="item.class">
+      <a :href="item.url" :target="item.target" rel="noreferrer">
+        <div class="card-content">
+          <div class="media">
+            <div v-if="item.logo" class="media-left">
+              <figure class="image is-48x48">
+                <img :src="item.logo" :alt="`${item.name} logo`" />
+              </figure>
+            </div>
+            <div v-if="item.icon" class="media-left">
+              <figure class="image is-48x48">
+                <i style="font-size: 35px" :class="['fa-fw', item.icon]"></i>
+              </figure>
+            </div>
+            <div class="media-content">
+              <p class="title is-4">{{ item.name }}</p>
+              <p class="subtitle is-6">{{ item.subtitle }}</p>
+            </div>
+            <div class="notifs">
+              <strong
+                v-if="activity > 0"
+                class="notif activity"
+                title="Activity"
+              >{{ activity }}</strong>
+              <strong
+                v-if="warnings > 0"
+                class="notif warnings"
+                title="Warning"
+              >{{ warnings }}</strong>
+              <strong
+                v-if="errors > 0"
+                class="notif errors"
+                title="Error"
+              >{{ errors }}</strong>
+              <strong
+                v-if="serverError"
+                class="notif errors"
+                title="Connection error to Radarr API, check url and apikey in config.yml"
+              >?</strong>
+            </div>
+          </div>
+          <div class="tag" :class="item.tagstyle" v-if="item.tag">
+            <strong class="tag-text">#{{ item.tag }}</strong>
+          </div>
+        </div>
+      </a>
+    </div>
+  </div>
+</template>
+
+<script>
+export default {
+  name: "Radarr",
+  props: {
+    item: Object,
+  },
+  data: () => {
+    return {
+      activity: null,
+      warnings: null,
+      errors: null,
+      serverError: false,
+    };
+  },
+  created: function () {
+    this.fetchConfig();
+  },
+  methods: {
+    fetchConfig: async function () {
+      this.health = await fetch(
```

Comment:
I would suggest to drop the async await here too. If you use await here, the next request will have to wait for this one to end before continuing. If you remove **all** the await, all requests will be executed simultaneously, reducing the loading time significantly!